arch/arm/soc/st_stm32/stm32f7/dts.fixup

```diff
@@ -1,5 +1,53 @@
 /* SoC level DTS fixup file */
 
-#define CONFIG_NUM_IRQ_PRIO_BITS				ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+#define CONFIG_NUM_IRQ_PRIO_BITS		ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define CONFIG_UART_STM32_PORT_1_BASE_ADDRESS	ST_STM32_USART_40011000_BASE_ADDRESS
+#define CONFIG_UART_STM32_PORT_1_BAUD_RATE	ST_STM32_USART_40011000_CURRENT_SPEED
+#define CONFIG_UART_STM32_PORT_1_IRQ_PRI	ST_STM32_USART_40011000_IRQ_0_PRIORITY
+#define CONFIG_UART_STM32_PORT_1_NAME		ST_STM32_USART_40011000_LABEL
+#define PORT_1_IRQ				ST_STM32_USART_40011000_IRQ_0
+
```

Comment:
I just gave it a try, and it doesn't seem to compile. It seems it should be renamed from `CONFIG_UART_STM32_PORT_1_` to `CONFIG_UART_STM32_USART_1_`

Reply:
I am using UART1 on STM32746G Discovery board. It compiles and works fine.

Reply:
As I said above, I tried your patchset rebased on master, that's why I have the issue and you don't. I have identified that it has been introduced by the following patch which has been recently committed: https://github.com/zephyrproject-rtos/zephyr/commit/fd26514a4d40bc54be40bb294bd03b105d674c40

You will need to fix this issue anyway before the patchset can be merged. The changes are as simple as replacing `_PORT_` by `_USART_` in `dts.fixup`.

Reply:
I will check it.

Reply:
I have rebased your patchset onto master and fixed this issue in the stm32f7 branch of my repository. The patch is available there: https://github.com/aurel32/zephyr/commit/1857f6628d6cdee3930d7aeb66d30b053f8d5e78

Please feel free to just take it and squash it into your commit adding `dts.fixup`.